Tools/cases_generator/formatting.py

```diff
@@ -58,13 +59,13 @@ def reset_lineno(self) -> None:
             self.set_lineno(self.lineno + 1, self.filename)
 
     @contextlib.contextmanager
-    def indent(self):
+    def indent(self) -> collections.abc.Iterator[None]:
```

Comment:
I still like `typing.Iterator` better, it's so much shorter!

Reply:
@AlexWaygood Play rock-scissors-paper with Guido

Reply:
Well, shorter yet is `from collections.abc import Iterator`!

Reply:
> Well, shorter yet is from collections.abc import Iterator!

Applied!